viewer/packages/pointclouds/src/annotationTypes.ts

```diff
@@ -34,7 +35,22 @@ export class BoxPrimitive {
 
 export type Geometry = CylinderPrimitive | BoxPrimitive;
 
```

Comment:
I think the number of types can be reduced here

Reply:
Removed the geometry types